README.md

```diff
@@ -34,6 +33,17 @@ Commonly used Gulp tasks shared across projects.
 - `gulp fractal:start`: Start Fractal server (if config path exists in `package.json`)
 - `gulp fractal:build`: Build static version of Fractal project
 
+## Options for js/css tasks
+
+Option | Effect
+:--- | :---
+`--minify` | Render additional minified files, suffixed with `.min`. Will be `true` in production builds.
+`--sourcemaps` | Render sourcemaps.
```

Comment:
```suggestion
`--sourcemaps` | Add source mappings to compiled files.
```
This should be automatically disabled for production builds. Is that the case already?  If so, then:
```suggestion
`--sourcemaps` | Add source mappings to compiled files. Disabled by default in production builds.
```